kuri_camera/scripts/ros_publisher_test.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```

Comment:
This and the other tests are good instruments to have available. Whenever you're ready to merge, please add a note to the readme describing their purpose.

Reply:
Haha these were actually just files for me to play around with different code, they aren't actually test cases. I've removed them